views/beneficial-owner-other.html

```diff
@@ -44,27 +45,30 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">Tell us about the corporate benefi
           label: {
             html: 'Building and street <span class="govuk-visually-hidden">line 1 of 2</span>'
           },
-          id: "address-line-1",
-          name: "address-line-1",
-          autocomplete: "address-line1"
+          id: "principalAddressLine1",
+          name: "principalAddressLine1",
+          value: principalAddress.principalAddressLine1 if principalAddress,
+          autocomplete: "principalAddressline1"
         }) }}
 
         {{ govukInput({
           label: {
             html: '<span class="govuk-visually-hidden">Building and street line 2 of 2</span>'
           },
-          id: "address-line-2",
-          name: "address-line-2",
-          autocomplete: "address-line2"
+          id: "principalAddressLine2",
+          name: "principalAddressLine2",
+          value: principalAddress.principalAddressLine2 if principalAddress,
+          autocomplete: "principalAddressline2"
         }) }}
 
         {{ govukInput({
           label: {
             text: "Town or city"
           },
           classes: "govuk-!-width-two-thirds",
-          id: "address-town",
-          name: "address-town",
+          id: "principalAddressTown",
+          name: "principalAddressTown",
+          value: principalAddress.principalAddressTown if principalAddress,
           autocomplete: "address-level2"
```

Comment:
A general question really (again, also pondered this with my change) - should autocomplete value match the 'name' field? You did this above for address lines1 and 2 but not here. I know that prototype template code seems to be inconsistent here too.

Reply:
Updated this.